lmn/templates/lmn/artists/artist_list.html

```diff
@@ -16,16 +16,47 @@ <h2 id='artist_list_title'>Artists matching '{{ search_term }}' <a href="{% url
   <h2 id='artist_list_title'>All artists</h2>
 {% endif %}
 
-
+<!--Table to display artist list-->
+<table class="table table-striped">
+  <thead>
+    <tr>
+      <th> ARTIST</th>
+      <th>See venues played, notes, and add your own </th>
+    </tr>
+  <tbody>
 {% for artist in artists %}
-
 <div class="artist" id="artist_{{ artist.pk }}">
-<P><a href="{% url 'lmn:artist_detail' artist_pk=artist.pk %}">{{ artist.name }}</a></p>
-  <p>See venues played, notes, and add your own <a href="{% url 'lmn:venues_for_artist' artist_pk=artist.pk %}">{{ artist.name }} notes</a>
-<div>
-
+  <tr>
+    <td><a href="{% url 'lmn:artist_detail' artist_pk=artist.pk %}">{{ artist.name }}</a></td>
+    <td><a href="{% url 'lmn:venues_for_artist' artist_pk=artist.pk %}">{{ artist.name }} notes</a></td>
+  </tr>
 {% empty %}
-<P>No artists found</P>
+<tr><td colspan="2">No artists found</td></tr>
 {% endfor %}
+</tbody>
+</table>
+<div>
 
+{% if artists.has_other_pages %}
+<!--Pagination if page has more list than specified for display in the views-->
```

Comment:
Good use of notes! Is it the ''' that allows sphynx to create documentation?

Reply:
If I would have waited I would have seen farther down in your code the answer to my question.